ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Rectangle selection/rectangle-selection-tool.spec.ts

```diff
@@ -0,0 +1,160 @@
+import { Page, test } from '@playwright/test';
+import {
+  BondTool,
+  BondType,
+  TopPanelButton,
+  clickOnAtom,
+  dragMouseTo,
+  drawBenzeneRing,
+  getControlModifier,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  getCoordinatesTopAtomOfBenzeneRing,
+  openFileAndAddToCanvas,
+  selectNestedTool,
+  selectTopPanelButton,
+  takeEditorScreenshot,
+  waitForIndigoToLoad,
+} from '@utils';
+import { getAtomByIndex } from '@utils/canvas/atoms';
+import { getBondByIndex } from '@utils/canvas/bonds';
+
+test.describe('Rectangle selection tool', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+    await waitForIndigoToLoad(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  const xDelta = 30;
+  const yDelta = 60;
+  const modifier = getControlModifier();
+
+  const xMark = 300;
+  const yMark = 200;
+
+  async function selectObjects(page: Page, xAxis: number, yAxis: number) {
+    const point = await getCoordinatesOfTheMiddleOfTheScreen(page);
+    await page.mouse.move(point.x - xAxis, point.y - yAxis);
+    await page.mouse.down();
+    await page.mouse.move(point.x + xAxis, point.y + yAxis);
+    await page.mouse.up();
```

Comment:
I guess you can use function dragMouseTo

Reply:
when I use dragMouseTo something goes terribly wrong. Is it ok if I leave it as page.mouse.move?

Reply:
sure